src/app/[locale]/page.tsx

```diff
@@ -1,13 +1,14 @@
-import { Header } from "@/components/header";
 import { HeroBlock } from "@/components/home/hero-block";
+import { SearchBlock } from "@/components/home/search/search-block";
 
 export default function Home() {
   return (
-    <main className="max-w-maxDesktop mx-auto flex min-h-screen flex-col items-center border">
-      <div className="desktopEnd:px-0 w-full px-20">
-        <Header />
+    <main className="flex flex-col items-center">
+      <div className="w-full px-20 desktopEnd:px-0">
```

Comment:
Im curious what does this class `desktopEnd` do?

Reply:
Design is moving into the tablet and phone phase. (maybe I chose the wrong name)